alf/environments/suite_gym.py

```diff
@@ -100,6 +105,10 @@ def wrap_env(gym_env,
     for wrapper in gym_env_wrappers:
         gym_env = wrapper(gym_env)
 
+    # To apply channel_first transpose on gym (py) env
```

Comment:
Probably add a detailed comment saying that all gym wrappers assume "channels_last" but pytorch only supports "channels_first"

Reply:
done